tools/lib/gitlint-rules.py

```diff
@@ -0,0 +1,40 @@
+from typing import Text, List
+
+from os import path
+
+import gitlint
+from gitlint.rules import LineRule, RuleViolation, CommitMessageTitle
+
+CURRENT_PATH = path.dirname(path.abspath(__file__))
+ACCEPTABLE_WORDS_FILE = path.join(CURRENT_PATH, 'commit-msg-words')
+
+
+class ImperativeTitle(LineRule):
+    """ This rule will enforce that the first word in the commit message
+    title uses an imperative action verb found in the acceptable word list. """
+
+    name = 'title-imperative-mood'
+    id = 'Z1'
+    target = CommitMessageTitle
+
+    error_msg = ('Commit title should start with an imperative action verb '
+                 '(not "{word}"): "{title}"')
```

Comment:
I think it might be worth having the error message here explain that if this lint rule seems to be in error, to consider modifying `tools/lib/commit-msg-words`.